hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/utils/TestMetadataStore.java

```diff
@@ -166,12 +170,12 @@ public void testIterator() throws Exception {
   public void testMetaStoreConfigDifferentFromType() throws IOException {
 
     OzoneConfiguration conf = new OzoneConfiguration();
-    conf.set(OzoneConfigKeys.OZONE_METADATA_STORE_IMPL, storeImpl);
+
     String dbType;
     GenericTestUtils.setLogLevel(MetadataStoreBuilder.LOG, Level.DEBUG);
     GenericTestUtils.LogCapturer logCapturer =
         GenericTestUtils.LogCapturer.captureLogs(MetadataStoreBuilder.LOG);
-    if (storeImpl.equals(OzoneConfigKeys.OZONE_METADATA_STORE_IMPL_LEVELDB)) {
+    if (storeImpl.equals(CONTAINER_DB_TYPE_LEVELDB)) {
```

Comment:
The logic needs to be reversed, maybe a typo.

Reply:
Thanks. Will fix it.